faunadb/client_test.go

```diff
@@ -2147,6 +2147,17 @@ func (s *ClientTestSuite) TestEchoAnObjectBack() {
 	s.Require().Equal(map[string]string{"key": "value"}, obj)
 }
 
+//see TestStringifyFaunaValues for more
+func (s *ClientTestSuite) TestStringifyQueryV() {
+	var q f.QueryV
+
+	query := f.Query(f.Lambda("X", f.Var("X")))
+	expected := `QueryV{Obj{"lambda": "X", "expr": Obj{"var": "X"}}}`
```

Comment:
Whats is the goal here? The string representation in this case is syntactically incorrect. It's not possible to create a `QueryV` with a `Obj` as rendered.

Reply:
Yes, that's correct. But at the same time its not possible to create a QueryV. I applied the change assuming it was raised  as a readability fix. 

cc: @nathaniel-may @faunaee

Reply:
Although readable, it's incorrect. I think this will be confusing to go developers.

Reply:
How about we render it as a function call instead? The `QueryV` is supposed to be opaque and not constructed by users anyway.